src/App.js

```diff
@@ -0,0 +1,20 @@
+import logo from "./logo.svg";
+import Form from "./components/Form";
+import Card from "./components/Card";
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+
+function App() {
+  return (
+    <div className="App ">
+      <BrowserRouter>
+        <Routes>
+          <Route path="/" element={<Card />} />
```

Comment:
`Card` name doesn’t justify name for the `home` route. Please rename this component to a meaningful name.
Same goes for `new`, and also change route name for `new`. It should be `add`/`create` or something like that.